tests/helpers.jsx

```diff
@@ -63,11 +72,27 @@ module.exports = {
     function RouterStub() {}
 
     assign(RouterStub, {
-      transitionTo: function() {},
+        makePath () {},
```

Comment:
Hmm...what's the point in stubbing these out? Do they all have default behavior you want to override?